pkgs/applications/virtualization/virtualbox/default.nix

```diff
@@ -7,7 +7,7 @@
 , javaBindings ? true, jdk ? null # Almost doesn't affect closure size
 , pythonBindings ? false, python3 ? null
 , extensionPack ? null, fakeroot ? null
-, pulseSupport ? config.pulseaudio or stdenv.isLinux, libpulseaudio ? null
+, pulseSupport ? false, libpulseaudio ? null
```

Comment:
Was this change intentionally?

Reply:
Right - if `config.pulseaudio` is false, we'd previously build a non-pulseaudio variant.

What about including `libpulseaudio` in `buildInputs` if `headless` is `true`? I doubt we save that much by building non-pulseaudio, but graphical virtualbox.

Reply:
updated.